WhaleSpotting/react-app/src/components/Button.tsx

```diff
@@ -38,7 +38,7 @@ export function Button({ style, text, onClick, dataTestId, minWidth25 = false, l
         return (
             <Link
                 data-testid={dataTestId}
-                className={`${styleClass} ${width} link`}
+                className={`${styleClass} ${width} link ${hidden ? "hidden" : "" }`}
```

Comment:
We don't need the classname afaik, here and below

Reply:
Addressed.